files/project_team/IFCTunnelDep3_Geo_BookA.ifc

```diff
@@ -0,0 +1,663 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X4NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'IFCTunnelDep3_Geo_BookA.ifc',
+/* time_stamp */ '2023-09-18T08:44:58',
+/* author */ ('lars'),
+/* organization */ ('Unknown'),
+/* preprocessor_version */ 'GeometryGymIFCcore v0.2.14.0 by Geometry Gym Pty Ltd built 2023-02-27T22:50:52',
+/* originating_system */ 'IFCTunnelDep3 v1.0.0.0',
+/* authorization */ 'None'
+);
+
+FILE_SCHEMA (('IFC4X4_75858E21'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCSITE('3m6veJZyzEnvqyL42iCjWt',#10,'My site',$,$,#646,$,$,$,$,$,$,$,$);
+#5= IFCPERSON('lars',$,$,$,$,$,$,$);
+#6= IFCORGANIZATION($,'Unknown',$,$,$);
+#7= IFCPERSONANDORGANIZATION(#5,#6,$);
+#8= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#9= IFCAPPLICATION(#8,' v1.0.0.0','IFCTunnelDep3 v1.0.0.0','IFCTunnelDep3 v1.0.0.0');
+#10= IFCOWNERHISTORY(#7,#9,$,.ADDED.,1695019498,$,$,1695019498);
+#11= IFCTUNNEL('3b29IIk6zBqhFfHxN6IdsJ',#10,'My Tunnel',$,$,#15,$,$,$,.ROAD.);
+#12= IFCRELAGGREGATES('301QLuz4MK99bi7_MSiykI',#10,$,$,#4,(#11));
+#13= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#14= IFCAXIS2PLACEMENT3D(#13,$,$);
+#15= IFCLOCALPLACEMENT(#3,#14);
+#16= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#17= IFCAXIS2PLACEMENT3D(#16,$,$);
+#18= IFCLOCALPLACEMENT(#15,#17);
+#19= IFCTUNNELPART('08EuEgfTD24f$YcMuMuM0C',#10,$,$,$,#18,$,$,$,.LONGITUDINAL.,.TUNNELSECTION.);
+#20= IFCRELAGGREGATES('0vY2M_w8Px7_dHfSh1FV2x',#10,$,$,#11,(#19,#24));
+#21= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#22= IFCAXIS2PLACEMENT3D(#21,$,$);
+#23= IFCLOCALPLACEMENT(#15,#22);
+#24= IFCTUNNELPART('0maeFFN4nAbQZ_fu0oQmhR',#10,$,$,$,#23,$,$,$,.LONGITUDINAL.,.TUNNELSECTION.);
+#25= IFCALIGNMENT('2bR9MSQg55_u5Mu39yZtYy',$,'E, oID:2','315',$,#3,#561,$);
+#26= IFCPROPERTYSINGLEVALUE('Status',$,IFCLABEL('NEW'),$);
+#27= IFCPROPERTYSET('0yrexkscj4he_NDzXyvi9o',$,'Pset_CivilElementCommon',$,(#26));
+#28= IFCRELDEFINESBYPROPERTIES('2pjP5eZcTAEPUqj3$0F90N',#10,$,$,(#25),#27);
+#29= IFCRELNESTS('2vE9Xt$jrEdevdYsmD4rbn',$,'name',$,#25,(#30,#214,#225,#236,#247,#258));
+#30= IFCREFERENT('1IfuWzQPz0nQRYIPoAzs8g',$,'Start station',$,$,#34,#203,.STATION.);
+#31= IFCPROPERTYSINGLEVALUE('Station',$,IFCLENGTHMEASURE(25980.799746),$);
+#32= IFCPROPERTYSET('3NzBVAodLERgps8r2hc97n',$,'Pset_Stationing',$,(#31));
+#33= IFCRELDEFINESBYPROPERTIES('2Swdj65SPC7vBZYneYNCGi',#10,$,$,(#30),#32);
+#34= IFCLINEARPLACEMENT(#3,#38,$);
+#36= IFCAXIS2PLACEMENT3D(#37,$,$);
+#37= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#38= IFCAXIS2PLACEMENTLINEAR(#39,$,$);
+#39= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(0.0),$,$,$,#40);
```

Comment:
```suggestion
#39= IFCPOINTBYDISTANCEEXPRESSION(IFCLENGTHMEASURE(0.0),$,$,$,#40);
```

I believe this will need to be updated at some point. (presuming 4x4 builds upon 4x3)